### DISCUSSION TITLE
Hetzner Cloud

### DISCUSSION THREAD

**Discussion by VarunAgw** [VarunAgw/CloudPingTest#1] [Ideas] [answered]
Add hetzner.cloud

**Comment by MusiCode1** [comment]
Here are the hetzner endpoints:
Helsinki: http://hel.icmp.hetzner.com
Falkenstein: http://fsn.icmp.hetzner.com
Ashburn: http://ash.icmp.hetzner.com

Source: https://www.reddit.com/r/hetzner/comments/euh3pe/hetzner_latency/

**Comment by VarunAgw** [comment, marked as answer]
Added :)

**Comment by jairantoniosantos** [comment]
Create - Italy North (Turin), "Turin", "eu-turin-1"
